content/docs/other-guides/istio-in-kubeflow.md

```diff
@@ -0,0 +1,96 @@
++++
+title = "Istio usage in Kubeflow"
```

Comment:
Change "usage" to "Usage".

(Our style guide is a little inconsistent, in that we use title case for page titles, and sentence case for headings within the page.)

Reply:
done